docs/src/modules/components/ApiPage/table/SlotsTable.tsx

```diff
@@ -106,6 +107,15 @@ export default function SlotsTable(props: SlotsTableProps) {
                       __html: description || '',
                     }}
                   />
+                  {tutorial && (
+                    <React.Fragment>
```

Comment:
I think using "<>" is much better and shorter then using React.Fragment
What do you think ?